package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
i think the body-parser needs to be upgraded too since it is using affected qs version https://github.com/expressjs/body-parser/blob/1.18.2/package.json#L19

Reply:
Thanks Ellen! body-parser is updated now too, 1.19.0 uses an updated version of qs: https://github.com/expressjs/body-parser/blob/1.19.0/package.json#L19
Talked with @mnightwardskytap and we think it's ok to update this in the same PR and under the same version update as express.

Reply:
thanks for the update!!